melos.yaml

```diff
@@ -1,4 +1,5 @@
 name: at_widgets
 
 packages:
-- packages/**
+  - packages/*
+  - packages/*/*example
```

Comment:
Explicitly identified the packages and examples just in case there are further nested pubspec.yaml files which we might want to ignore in melos